projects/plugins/jetpack/modules/widgets/eu-cookie-law.php

```diff
@@ -179,8 +179,21 @@ public function widget( $args, $instance ) {
 				$classes['hide'] = 'hide-on-button';
 			}
 
+			/**
+			 * Check if widget is loaded in widgets.php.
+			 *
+			 * @return string widget Static version of the widget for better preview.
+			 */
+			global $pagenow;
+			if ( 'widgets.php' === $pagenow ) {
+				echo '<div id="eu-cookie-law" style="padding: 0;margin: 5px">';
+				require_once __DIR__ . '/eu-cookie-law/widget.php';
+				echo '</div>';
+				return;
+			}
+
 			echo $args['before_widget'];
-			require( dirname( __FILE__ ) . '/eu-cookie-law/widget.php' );
+			require_once __DIR__ . '/eu-cookie-law/widget.php';
 			echo $args['after_widget'];
 			/** This action is already documented in modules/widgets/gravatar-profile.php */
 			do_action( 'jetpack_stats_extra', 'widget_view', 'eu_cookie_law' );
```

Comment:
Maybe those args and the hook should also be triggered in the widget editor, for consistency?

Reply:
The args are what makes it a pop-up. It wraps it in a div that receives all the styling to make it float. That's why I replaced it with the custom div instead.

If there was a way to filter out the args and remove that piece? Or is there another option you think would be better? I tried making styling changes that superceded the position but it never worked.

Reply:
ah right, I missed that.  Maybe expand on the inline comment you added so we remember all this 6 months from now?

Reply:
Cool, updated!